html/pages/device.inc.php

```diff
@@ -229,6 +229,14 @@
                 </a>
               </li>');
 
+        if (@dbFetchCell("SELECT COUNT(stp_id) FROM stp WHERE device_id = '".$device['device_id']."'") > '0') {
```

Comment:
Instead of `count` just select `1` - will take some load away from the sql if the table gets too populated